README.md

```diff
@@ -80,6 +80,18 @@ Check out the [sample][sample].
 Known Limitations
```

Comment:
Adding a comment here because apparently you can't comment on images. 

Are we trying to keep `five_bucks.png`?